index.html

```diff
@@ -0,0 +1,159 @@
+      </div>
+      <div class="intro-date">
+        <h2>2025.01.28 (TUE) ~ 2025.01.30 (THU) </h2>
+        <p>@ UCV University Stadium</p>
+      </div>
+    </section>
+
+    <section class="main-program">
+      <div class="main-program-title">
+          <h3>Main Program</h3>
+          <div class="orange-line"></div>
+      </div>
+      <div class="main-program-elements">
+          <div class="main-program-element">
+            <img src="images/main-one.png" alt=""/>
+            <h3>Lecture</h3>
+            <p>Hear from Professionals around the wordl about the cryptocurrincies and how it's gonne be the new money</p>
+          </div>
+          <div class="main-program-element mid-program">
+            <img src="images/main-two.png" alt=""/>
+            <h3>Cryptos Exhibition</h3>
+            <p>
+                Presentation of the best crypto projects and new interesting projects for this year</p>
+          </div>
+          <div class="main-program-element mid-program">
+            <img src="images/main-three.png" alt=""/>
+            <h3>Forum</h3>
+            <p>Have the time to share your thoughts and opinions with experts in each topics</p>
+          </div>
+          <div class="main-program-element mid-program">
+            <img src="images/main-four.png" alt=""/>
+            <h3>Trade</h3>
+            <p>Learn the basics to the advanced to be able to trade and obtain benefits with cryptos</p>
+          </div>
+          <div class="main-program-element mid-program">
+            <img src="images/main-five.png" alt=""/>
+            <h3>Crypto Share</h3>
+            <p>Get the opportunity to network with Crypto New Order affliates around the world and get news about the crpto world</p>
+          </div>
+          <div class="join-button">
+              <button type="button">Join to the future</button>
+          </div> 
+      </div>
+    </section>
+
+    <section class="speakers-div">
+      <div class="featured-speakers-title">
+          <h2>Featured Speakers</h2>
+          <div class="orange-line"></div>
+      </div>
+      <div class="featured-speakers"></div>
+      <div class="button-speaker">
+        <button class="more-button" type="button">MORE <span class="more-arrow">v</span></button>
+      </div>
+      <div class="other-speakers"></div>
+    </section>
+
+    <section class="partner-section">
+      <div class="partner-title">
+        <h2>Partner</h2>
+        <div class="orange-line"></div>
+      </div>
+      <div class="partners-logo">
+        <img src="images/binance-logo.png" alt="binance" id="binance">
+        <img src="images/crypto-com.png" alt="crypto.com" id="crypto">
+        <img src="images/coingecko-logo.png" alt="coingecko" id="coingecko">
+        <img src="images/coinmarketcap-logo.png" alt="coinmarketcap" id="coinmarketcap">
+        <img src="images/coinbase-logo.png" alt="coinbase" id="coinbase">
```

Comment:
- [ ] Please wrap these images in a list, You can use `<ul>.` When we similar elements on the page It is a good practice to wrap them in the list and it will also make sure the code is semantic.